types/superagent/index.d.ts

```diff
@@ -16,6 +16,10 @@ type CallbackHandler = (err: any, res: request.Response) => void;
 
 type Serializer = (obj: any) => string;
 
+type MultipartValueSingle = Blob | Buffer | fs.ReadStream | string | boolean;
```

Comment:
We can also add `number` type because `FormData.attach()` used in both `attach()` and `field()` will convert it to string.